editor/src/core/shared/element-path-tree.ts

```diff
@@ -94,7 +94,7 @@ function getChildrenPaths(
     element.element.value.children.length > 0
   ) {
     childrenFromElement = element.element.value.children
-      .filter((child) => !isJSXTextBlock(child) && !isJSExpressionOtherJavaScript(child))
+      .filter((child) => !isJSXTextBlock(child) && !isJSExpressionMapOrOtherJavaScript(child))
```

Comment:
I wonder if this condition should be an exhaustive switch/case so that any subsequent changes don't cause later pains.